src/bin/sage-update-version

```diff
@@ -87,6 +87,25 @@ EOF
 # Create a top-level VERSION.txt file, which some external utilities rely on
 echo "$SAGE_VERSION_BANNER" > "$SAGE_ROOT/VERSION.txt"
 
+# Add version to the front of GitHub release assets URLs.
+SAGE_MINOR_VERSION=${SAGE_VERSION//.alpha*/}
+SAGE_MINOR_VERSION=${SAGE_MINOR_VERSION//.beta*/}
+SAGE_MINOR_VERSION=${SAGE_MINOR_VERSION//.dev*/}
+SAGE_MINOR_VERSION=${SAGE_MINOR_VERSION//.post*/}
+SAGE_MINOR_VERSION=${SAGE_MINOR_VERSION//.rc*/}
+( echo "https://github.com/sagemath/sage/releases/download/$SAGE_MINOR_VERSION/"
+  sed '/^#/d' "$SAGE_ROOT/.upstream.d/20-github.com-sagemath-sage-releases"
+) | uniq | head -n 3 > "$SAGE_ROOT/.upstream.d/20-github.com-sagemath-sage-releases.tmp"
+( cat <<EOF
+# Upstream packages as uploaded as GitHub release assets.
+# This file is automatically updated by the sage-update-version script.
+EOF
+  cat "$SAGE_ROOT/.upstream.d/20-github.com-sagemath-sage-releases.tmp"
+) > "$SAGE_ROOT/.upstream.d/20-github.com-sagemath-sage-releases"
+rm -f "$SAGE_ROOT/.upstream.d/20-github.com-sagemath-sage-releases.tmp"
+
+exit
```

Comment:
Why `exit`? Doesn't that end the script before executing the rest of it? Also, do we need to `git commit` the new file?

Reply:
Whoops, that was debugging code